SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1172,7 +1182,11 @@ void USpatialActorChannel::ServerProcessOwnershipChange()
 
 void USpatialActorChannel::UpdateEntityACLToNewOwner()
 {
-	FString NewOwnerWorkerAttribute = SpatialGDK::GetOwnerWorkerAttribute(Actor);
+	FString NewOwnerWorkerAttribute;
+	{
+		SCOPE_CYCLE_COUNTER(STAT_GetOwnerWorkerAttribute);
```

Comment:
Move this inside `SpatialGDK::GetOwnerWorkerAttribute` ?

Reply:
Total nit, feel free to dismiss.

Reply:
makes sense, will do. My guess is it was done this way to minimize divergence on our end.

Reply:
On second thought... that would mean either moving the impl for GetOwnerWorkerAttribute out of the header, or adding an include for SpatialNetDriver in order to pick up the decl for the stat group. It's probably worth a discussion whether we want to toss all of these stats into that group at all, or should define new groups. Will leave this PR until we can chat in the new year.

Reply:
Merged in as-is; we can chat about the bigger picture and adjust later.